recipes/Functions.py

```
@@ -394,6 +406,32 @@ def called_fn(self) -> CalledFunction | None:
         )
 
 
+def _render_package_json_editor(*, label: str, help: str) -> None:
```

Ruff (0.15.20)

[error] Function argument `help` is shadowing a Python builtin

(A002)